src/exports/index.ts

```diff
@@ -5,14 +5,20 @@ export type {
   DeployOptions,
   DeploySentTx,
 } from "../contract-deploy.js";
+export { Contract } from "../contract.js";
 export type {
-  Contract,
   ContractFunctionInteraction,
   IntentAction,
   SendOptions,
 } from "../contract.js";
+export { createEip1193ProviderFromAccounts } from "../createEip1193ProviderFromAccounts.js";
```

Comment:
this is an example file. Do you need it in your project?

Reply:
have you tried this? https://github.com/nemi-fi/wallet-sdk?tab=readme-ov-file#convert-aztecjs-wallet-to-account